main.py

```diff
@@ -2502,7 +2504,8 @@ async def givecat(message: discord.Interaction, person_id: discord.User, amount:
         return
 
     add_cat(message.guild.id, person_id.id, cat_type, amount)
-    embed = discord.Embed(title="Success!", description=f"gave <@{person_id.id}> {amount} {cat_type} cats", color=0x6E593C)
+    icon = get_emoji(cattype.lower() + "cat")
```

Comment:
same thing here

Reply:
yeah, whoops, that's what happens when you copy-paste things without actually looking...
should we make instances either `cat_type` or `cattype` for consistency?